graphql/schema/schema.go

```diff
@@ -247,8 +247,8 @@ type Trace struct {
 }
 
 type TraceBrief struct {
-	Traces []*BasicTrace `json:"traces"`
-	Total  int           `json:"total"`
+	Data  []*BasicTrace `json:"data"`
```

Comment:
Do we need the field name to be `Data` as well? or can it be 

```go
Traces []*BasicTrace `json:"data"`
```

the name `Traces` may be more semantic

Reply:
ok